lib/Dialect/FIRRTL/FIRRTLTypes.cpp

```diff
@@ -562,7 +582,7 @@ FIRRTLBaseType FIRRTLBaseType::getPassiveType() {
 FIRRTLBaseType FIRRTLBaseType::getConstType(bool isConst) {
   return TypeSwitch<FIRRTLBaseType, FIRRTLBaseType>(*this)
       .Case<ClockType, ResetType, AsyncResetType, AnalogType, SIntType,
-            UIntType, BundleType, FVectorType, FEnumType>(
+            UIntType, BundleType, FVectorType, FEnumType, BaseTypeAliasType>(
```

Comment:
Is there a reason why members like `getConstType` and `getPassiveType` are defined while othres like `getMaskType` and `getBitWidthOrSentinel` are not defined under `BaseAliasType` ?

Reply:
I followed the same convention of FVectorType or BundleType I'm not sure why there is difference, but I agree that that's more consistent. I actually hope there is a type trait  for FIRRTLBaseType and every type defines these helpers as member function, but I think we should do a different PR.